modules/Vtiger/models/MiniList.php

```diff
@@ -54,12 +54,21 @@ public function getTargetModule()
 		return $this->extraData['module'];
 	}
 
-	public function getTargetFields()
+	public function getTargetFields($extraField = false)
 	{
 		$fields = $this->extraData['fields'];
+		$moduleName = $this->getTargetModule();
 		if (!\in_array('id', $fields)) {
 			$fields[] = 'id';
 		}
+		if ('Calendar' === $moduleName && $extraField) {
+			$moduleModel = \Vtiger_Module_Model::getInstance($moduleName);
+			if (\in_array('date_start', $fields) && ($fieldModel = \Vtiger_Field_Model::getInstance('time_start', $moduleModel)) && $fieldModel->isActiveField() && $fieldModel->isViewable()) {
+				$fields[] = 'time_start';
+			} elseif (\in_array('due_end', $fields) && ($fieldModel = \Vtiger_Field_Model::getInstance('time_end', $moduleModel)) && $fieldModel->isActiveField() && $fieldModel->isViewable()) {
```

Comment:
You cannot use elseif
```suggestion
			}
			if (\in_array('due_end', $fields) && ($fieldModel = \Vtiger_Field_Model::getInstance('time_end', $moduleModel)) && $fieldModel->isActiveField() && $fieldModel->isViewable()) {
```